terraform-sentinel/step1.md

```diff
@@ -0,0 +1,90 @@
+In this scenario, you will apply the Sentinel logic principles to a Terraform specific deployment.
+
+Navigate to the `terraform-sentinel/tf-config/main.tf`{{open}} and review the configuration you are testing.
+
+This configuration builds an S3 bucket with a unique name and deploys an example web app as a bucket object. You have an S3 bucket policy attached to the bucket resource which allows pubic read permissions for your bucket object. This example configuration does not have any deployment safeguards built in and if your AWS user has S3 build and upload permissions, your Terraform deployment will apply successfully.
```

Comment:
```suggestion
This configuration builds a publicly-readable S3 bucket with a unique name and deploys an example web app as a bucket object. Noice the `policy` block of the `"aws_s3_bucket" "bucket"` resource that makes the bucket readable. 

This example configuration does not have any deployment safeguards built in and if your AWS user has S3 build and upload permissions, your Terraform deployment will apply successfully, making the contents of the bucket public.
```

Reply:
Just realized that the policy is testing for tags, not whether or not the bucket is public 🤦‍♀️. We should probably mention that somewhere in here and highlight it in the configuration?